### ISSUE TITLE
no-restricted-tags does not work with example tags

### ISSUE THREAD

**Issue by xgbuils** [vsiakka/gherkin-lint#163]
If I forbide the tags `@exampletag1` and `@exampletag2` with `no-restricted-tags` rule.

``` gherkin
Scenario Outline: This is a Scenario Outline with two duplicate tags
  Then this is a then step <foo>
@exampletag1
Examples:
  | foo |
@exampletag2
Examples:
  | bar |
```

errors are not reported.
